jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1437,28 +1433,11 @@ public static void appendSchemeHostPort(StringBuilder url, String scheme, String
      */
     public static void appendSchemeHostPort(StringBuffer url, String scheme, String server, int port)
```

Comment:
Can we deprecate this StringBuffer version? (just because it's StringBuffer)

Reply:
It is used in Servlet API which still exposed StringBuffer.   But I guess we could to discourage its usage.